src/tui.rs

```diff
@@ -403,6 +409,19 @@ impl ToipeTui {
         //   - won't hang there waiting for user to type space
         lines.push(Text::from(line.join(" ")).with_faint());
 
+        if lines.len() + self.bottom_lines_len + 1 >= terminal_height as usize {
+            return Err(ToipeError::from(format!(
+                "Terminal height is too short! Toipe requires at least {} lines, got {} lines",
+                lines.len() + self.bottom_lines_len + 1,
+                terminal_height,
+            )));
+        } else if max_word_len >= max_width as usize {
+            return Err(ToipeError::from(format!(
+                "Terminal width is too low! Toipe requires at least {} columns, got {} columns",
```

Comment:
Same here. When `max_word_len == max_width`, the error isn't right

Reply:
done